src/Microsoft.AspNet.Mvc.Core/DependencyInjection/MvcCoreMvcCoreBuilderExtensions.cs

```diff
@@ -62,7 +62,7 @@ public static IMvcCoreBuilder AddFormatterMappings(this IMvcCoreBuilder builder)
         // Internal for testing.
         internal static void AddFormatterMappingsServices(IServiceCollection services)
         {
-            services.TryAddTransient<FormatFilter, FormatFilter>();
+            services.TryAddSingleton<FormatFilter, FormatFilter>();
```

Comment:
Curious - why is this service not registered as the `IFormatFilter` interface? #n00b